spec/unit/blockscore/candidate_spec.rb

```diff
@@ -32,7 +32,7 @@ module BlockScore
       subject(:search) { -> { candidate.search(constraints) } }
 
       context 'search request' do
-        let(:uri)      { '/watchlists'                                         }
+        let(:uri)      { "/watchlists"                                         }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.